Sources/AblyAssetTrackingPublisher/DefaultPublisher.swift

```diff
@@ -67,7 +71,9 @@ class DefaultPublisher: Publisher {
          ablyPublisher: AblyPublisher,
          locationService: LocationService,
          routeProvider: RouteProvider,
-         enhancedLocationState: TrackableState<EnhancedLocationUpdate> = TrackableState()
+         enhancedLocationState: TrackableState<EnhancedLocationUpdate> = TrackableState(),
+         rawLocationState: TrackableState<RawLocationUpdate> = TrackableState(),
+         areRawLocationsEnabled: Bool? = nil
```

Comment:
Isn't it just yes/no? Why optional used?

Reply:
taken from android implementation, don't know if "third" value (the `nil`) will be used in future? @KacperKluka ?

Reply:
I don't remember exactly but it was probably connected with backwards compatibility :thinking:

Reply:
Is @KacperKluka response is enough to close this conversation?